_includes/css/just-the-docs.scss.liquid

```diff
@@ -3,5 +3,8 @@ $logo: "{{ site.logo | absolute_url }}";
 {% endif %}
 @import "./support/support";
 @import "./color_schemes/{{ include.color_scheme }}";
+{%- if include.dark_color_scheme and include.dark_color_scheme != include.color_scheme %}
+@import "./color_schemes/{{ include.dark_color_scheme }}";
+{%- endif %}
```

Comment:
I wasn't paying much attention to what this file was used for - I think this is not needed.

Reply:
Yes, it's included to make the color files. Dropped.